website/docs/r/iothub_digital_twins.html.markdown

```diff
@@ -0,0 +1,70 @@
+---
+subcategory: "IoT Hub"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_iothub_digital_twins"
+description: |-
+  Manages a Digital Twins.
+---
+
+# azurerm_iothub_digital_twins
+
+Manages a Digital Twins.
```

Comment:
Digital Twin what? instance? something else?

Reply:
Fixed in PR #9430